rules/windows/lateral_movement_incoming_winrm_shell_execution.toml

```diff
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2020/11/24"
+maturity = "production"
+updated_date = "2020/11/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies remote execution via Windows Remote Management (WinRM) Remote Shell on the target host. This could be an
+indication of lateral movement.
```

Comment:
```suggestion
Identifies remote execution via Windows Remote Management (WinRM) remote shell on a target host. This could be an
indication of lateral movement.
```

Reply:
may need to re-lint